### ISSUE TITLE
Changing comments.css does not change appearance of comment area.

### ISSUE THREAD

**Issue by EdHamilton9** [jacobwb/hashover-next#232]
Yesterday, April 30, I downloaded hashover-next, unzipped it, and copied the files to my local server. The main hashover directory is directly underneath /var/www/public_html. (Debian sid, but I relocated the localhost to public_html.)

The commenting system works fine, but nothing I do to hashover/themes/default/comments.css changes the appearance. I would like to modify the colors to blend in with my site. Evenually I hope to put my changes in a separate .css file and call it after the other .css files. 

I am using the default theme:

line 33 of hashover/backend/classes/settings.php is
	public $theme			= 'default';			// Comment Cascading Style Sheet (CSS)
	
settings.php is executable:
	
[ed@sid public_html]$ ll hashover/backend/classes/settings.php 
-rwxr-xr-x 1 ed ed 13147 May  1 12:48 hashover/backend/classes/settings.php

comments.css is readable by all:

[ed@sid public_html]$ ll hashover/themes/default/comments.css 
-rw-r--r-- 1 ed ed 26186 May  1 16:42 hashover/themes/default/comments.css	

What am I missing. How do I change the background-color of hashover-inputs, for example?

Thank you, 
Ed H

**Comment by CyberShadow** [comment]
Do you see your changes in the CSS source if you look at http://localhost/hashover/themes/default/comments.css ?

**Comment by EdHamilton9** [comment]
no.

**Comment by CyberShadow** [comment]
Then (excluding some really unusual setups) you have multiple copies of the hashover source somewhere on the server - one being the one you're editing, and the other being the one served by your web server.

**Comment by EdHamilton9** [comment]
I'll take a look around. Before I start Googling, do you know if there is a quick way to tell which source is being served by my web server? I know which copy I am editing. It is /var/www/public_html/hashover/themes/default/comments.css

**Comment by CyberShadow** [comment]
It will be in the configuration, in one way or another. Some distributions have a really convoluted way of managing it, though.

**Comment by EdHamilton9** [comment]
oops. I refreshed http://localhost/hashover/themes/default/comments.css in the browser, and I <i>can</i> see the changes I have made.

**Comment by CyberShadow** [comment]
Do you see the changes on the page now?

**Comment by EdHamilton9** [comment]
yes, and I have no idea what I did differently this time. I'm not an expert, but not exactly new at hacking css either. I did it for my online website a lot.
I've been at this a few hours, and it is the first time it worked. I refreshed the page each time, but haven't seen changes until now. Thank you for your help and patience.

I really am impressed with hashover. I looked around a lot for a commenting system that worked on static pages and did not like most of what I saw. I was going to try Isso, but I don't self-host my site. It is web-hosted, and Isso was not one of the apps I could install through Cpanel.

**Comment by EdHamilton9** [comment]
hashover/backend/classes/formui.php

I think what I did was change something in hashover/backend/classes/formui.php by accident, while looking for the problem.

While I was waiting for a reply to my original question, I noticed that my working copy of formui.php was different from the original. So I copied the original into my hashover file on my local site.

That was right before the .css worked for the first time.

Unfortunately for troubleshooting, I can't find a bad copy of the file, so I don't know what I changed.

I will mark the issue closed. Thank you again.
